Improve git integration documentation (#729)
zeit/docs: pages/docs/v2/integrations/now-for-github.mdx

}
```

If you decide to revert a commit that has already been aliased, the previous commit's deployment will automatically be aliased instantly; providing you with instant rollbacks.

### Staging Aliases for the Latest Changes for Each Pull Request

> The latest push to any pull request will automatically be aliased to a unique URL based on the project name, branch, and team or username. These URLs will be given See more on automatic aliasing for git pushes and beyond in the [Aliasing Documentation](/docs/v2/domains-and-aliases/aliasing-a-deployment/#git-staging-aliases).

Reviewer: This doesn't look quite right - 'These URLs will be given See more...'

### Deployment Authorizations for Forks

> If you recieve a pull request from a fork of your repository and there is either a change to the [`now.json`](/docs/v2/deployments/configuration) file or there are [secrets](/docs/v2/deployments/environment-variables-and-secrets/#securing-environment-variables-using-secrets) used in your project; Now will require an authorization from your or a member of your team to deploy the pull request.

Reviewer: ```suggestion
If you recieve a pull request from a fork of your repository and there is either a change to the [`now.json`](/docs/v2/deployments/configuration) file or there are [secrets](/docs/v2/deployments/environment-variables-and-secrets/#securing-environment-variables-using-secrets) used in your project; Now will require an authorization from you or a member of your team to deploy the pull request.
```

> This behavior protects you and your project from a leak of sensistive information.

Reviewer: ```suggestion
This behavior protects you and your project from a leak of sensitive information.
```

## Extending Your GitHub Workflow

> Now for GitHub uses the deployment API to bring you an extended user interface both in GitHub, when showing deployments, and if you have notifications setup in Slack using the [Slack GitHub app](https://slack.github.com).

Reviewer: ```suggestion
Now for GitHub uses the deployment API to bring you an extended user interface both in GitHub, when showing deployments, and Slack, if you have notifications setup using the [Slack GitHub app](https://slack.github.com).
```

You will see all of your deployments, production or staging, from within GitHub in its own page. See the page for this very documentation: https://github.com/zeit/docs/deployments

> Due to using GitHub's Deployments API, you will also be able to integrate with other services through [GitHub's checks](https://help.github.com/en/articles/about-status-checks). Now for GitHub will provide the deployment URL to the checks that require it, for example; to a testing a suite such as [Ghost Inspector](https://ghostinspector.com/docs/integration/github/).

Reviewer: ```suggestion
Due to using GitHub's Deployments API, you will also be able to integrate with other services through [GitHub's checks](https://help.github.com/en/articles/about-status-checks). Now for GitHub will provide the deployment URL to the checks that require it, for example; to a testing suite such as [Ghost Inspector](https://ghostinspector.com/docs/integration/github/).
```

<Image
  src={`${process.env.ASSETS}/docs/now-for-github/github-checks.png`}
  width={1738/2}
  height={1046/2}
/>

> <Caption style={{marginTop: '-40px'}}>A deployment from a Now for GitLab push.</Caption>

Reviewer: ```suggestion
<Caption style={{marginTop: '-40px'}}>A deployment from a Now for GitHub push.</Caption>
```

Author: Ouch, good catch, thanks! It should be, however
```suggestion
<Caption style={{marginTop: '-40px'}}>The checks UI from GitHub containing Now and Ghost Inspector.</Caption>
```
